### ISSUE TITLE
Follow-up update of go-dvote and multi-rpc

### ISSUE THREAD

**Issue by emmdim** [vocdoni/vocdoni-manager-backend#36]
Blocked by https://github.com/vocdoni/vocdoni-node/issues/155

**Comment by emmdim** [comment]
solved by bb0ddbbbf21ea34db75950ac4bbe4a0769000f22

**Comment by mvdan** [comment]
@emmdim pinging this thread as requested - but I see you're already done :)
